src/WebJobs.Script.WebHost/Controllers/FunctionsController.cs

```diff
@@ -74,11 +74,19 @@ public async Task<IActionResult> CreateOrUpdate(string name, [FromBody] Function
 
             bool success, configChanged;
             FunctionMetadataResponse functionMetadataResponse;
-            using (fileMonitoringService.SuspendRestart())
+            using (fileMonitoringService?.SuspendRestart())
```

Comment:
Have you seen instances where this was null?

Reply:
No, actually. Will remove "?".